scene/3d/physics_body_3d.cpp

```diff
@@ -3251,6 +3259,15 @@ Vector3 PhysicalBone3D::get_joint_rotation() const {
 	return joint_offset.basis.get_euler_normalized();
 }
 
+void PhysicalBone3D::set_joint_exclude_nodes_from_collision(bool p_exclude_nodes_from_collision) {
+	joint_exclude_nodes_from_collision = p_exclude_nodes_from_collision;
```

Comment:
```suggestion
	if (joint_exclude_nodes_from_collision == p_exclude_nodes_from_collision) {
		return;
	}
	
	joint_exclude_nodes_from_collision = p_exclude_nodes_from_collision;
```
Prevent unnecessary update if the value does not change.

Reply:
Changed that, thanks!